packages/maskbook/src/_locales/en/messages.json

```
@@ -524,8 +524,8 @@
     "plugin_ito_loading": "Loading ITO ...",
     "plugin_ito_loading_failed": "Loading failed",
     "plugin_ito_loading_try_again": "Try Again",
-    "plugin_ito_amount_approved": "Approved {{amount}} {{symbol}}",
-    "plugin_ito_amount_approved_infinity": "Approved infinity {{symbol}}",
+    "plugin_ito_amount_approved": "Unlocked {{amount}} {{symbol}}",
```

```suggestion
    "plugin_ito_amount_unlocked": "Unlocked {{amount}} {{symbol}}",
```

plz sync update i18n key name.